pansat/download/commandline.py

```diff
@@ -0,0 +1,161 @@
+"""
+pansat.download.commandline
+===============================
+The ``commandline´´ submodule allows to download data products from commandline using the argparse module.
+
+The following flags can be used:
+    --type
+    --pm
+    --product/-prod
+    --starttime/-t0
+    --endtime/-t1
+    --variable/-var
+    --domain/-d
+
+
+"""
+import argparse
+import datetime
+import pkgutil
+import importlib
+import sys
+import pansat.products
+
+
+def download():
```

Comment:
It's trivial but it would be nice if you cold add a docstring to the method.

Reply:
Good point, I added a docstring